tests/test_tk_fixtures.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tk_toolchain.testing import create_unique_name
+from tk_toolchain.authentication import get_toolkit_user
+
+
+def test_create_project(tk_test_shotgun, tk_test_create_project):
+    """
+    Ensure the project gets created successfully.
+    """
+    # Get project info
+    project_name = create_unique_name("Toolkit UI Automation")
+    filters = [["name", "is", project_name]]
+    existed_project = tk_test_shotgun.find_one("Project", filters)
+
+    # Get current project info
+    current_project = tk_test_create_project
+
+    # Make sure both file has the same id.
+    assert existed_project["id"] == current_project["id"]
+
+
+def test_credentials(tk_test_shotgun):
+    """
+    Ensure getting credentials
+    """
+    # Get credentials from the fixture and fin local storage with it
+    fixture_credentials = tk_test_shotgun
+
+    # Use credentials from fixture to get storage info
+    storage_name = create_unique_name("Toolkit UI Automation")
+    local_storage1 = fixture_credentials.find_one(
+        "LocalStorage", [["code", "is", storage_name]], ["code"]
+    )
+
+    # Getting credentials from TK_TOOLCHAIN
+    tk_toolchain_crendentials = get_toolkit_user().create_sg_connection()
+
+    # Use credentials from TK_TOOLCHAIN to get storage info
+    local_storage2 = tk_toolchain_crendentials.find_one(
+        "LocalStorage", [["code", "is", storage_name]], ["code"]
+    )
+
+    # Make sure both storage info are identical.
+    assert local_storage1 == local_storage2
+
+
+def test_current_user(tk_test_shotgun, tk_test_current_user):
+    """
+    Ensure getting current_user
+    """
+    # Getting current user
+    user = get_toolkit_user()
+    username = tk_test_shotgun.find_one(
+        "HumanUser", [["login", "is", str(user)]], ["name"]
```

Comment:
Actually, you should be getting the user name of the `tk_test_shotgun.config.user_login`. You're trying to make sure that `get_toolkit_user` is getting it's user from the connection, not somewhere else, right?

Reply:
I want to make sure fixture `tk_test_current_user` (which returns the name of the user) is identical than the one from `get_toolkit_user`